### PR TITLE
Fixed issue 3186

### PR DESCRIPTION
**Pull request by angwangri**
Link for test: https://originprotocol.github.io/test-builds/angwangri/#/

### CODE REVIEW

File: dapps/marketplace/src/pages/listings/_Search.js [hunk 1]
```diff
@@ -260,16 +262,16 @@ require('react-styl')(`
...
         .featured-categories-wrapper
-          margin: 0 -2rem
-          overflow-x: scroll
+          margin: 0 0rem
 
         .featured-categories
           display: inline-flex
-          padding: 0 2rem
+          margin: 0 -10px
+          flex-wrap: wrap;
```
Comment (franckc) [inline dapps/marketplace/src/pages/listings/_Search.js]: nit: the convention in our codebase is to leave the semicolons (e.g. character ";") out. please remove.

**Comment by micahalcorn** [comment]
Much improved 👍 It would also be preferable for those filter icons (Rewards, Housing, etc) to be justified horizontally with equidistant horizontal margin as described [here](https://github.com/OriginProtocol/origin/issues/3186#issuecomment-535721320). I think I would also prefer the "Clear" to be right-aligned in that field on small screens instead of being padded.

**Comment by micahalcorn** [comment]
@franckc let me commit a small change and then I'll merge it. 🙂

**Comment by micahalcorn** [comment]
I've made a small change with 9f218f3. @angwangri will you make one more tweak to the margin/padding so that these icons light up with the nav controls?

Screen Shot 2019-09-29 at 12 52 06 PM

**Comment by micahalcorn** [comment]
@angwangri let us know if you update the test build again, please. 🏗

**Comment by angwangri** [comment]
@micahalcorn I fixed the issue. 
Here is the updated test-build.
https://originprotocol.github.io/test-builds/angwangri/#/

**Comment by micahalcorn** [comment]
I made another change in https://github.com/OriginProtocol/origin/pull/3526/commits/e6e4c71040dc6cb546d1e4130d5e3c42d1adb42f so that we don't end up with this 👇

Screen Shot 2019-10-01 at 10 57 32 AM

**Pull request merged**